src/PrestaShopBundle/Resources/views/Admin/Helpers/bootstrap_popup.html.twig

```diff
@@ -42,8 +42,8 @@
                     <div class="float-right progress-details-text" default-value="{{ progressbar.label|default('Processing...'|trans({}, 'Admin.Global')) }}">
                         {{ progressbar.label|default('Processing...'|trans({}, 'Admin.Global')) }}
                     </div>
-                    <div class="progress active progress-striped" style="display: block; width: 100%">
-                        <div class="progress-bar progress-bar-success" role="progressbar" style="width: 0%">
+                    <div class="progress" style="width: 100%">
+                        <div class="progress-bar progress-bar-striped" role="progressbar" style="width: 0%">
```

Comment:
Only if "success" green color was the initial intent :
```suggestion
                        <div class="progress-bar progress-bar-striped bg-success" role="progressbar" style="width: 0%">
```

Reply:
Strange thing, I don't remember if the default state was success, I don't think so, maybe there are some JS magic behind this

Reply:
We should use green color for 100% state. Blue color means "in progress". Btw progress bar is inside modal, and after is comleted, moldal will be closed.